site/guide/9-configuration.md

```diff
@@ -148,7 +152,8 @@ workers = 16
 max_blocking = 512
 keep_alive = 5
 ident = "Rocket"
-ip_header = "X-Real-IP" # set to `false` to disable
+ip_header = "X-Real-IP" # set to `false` or `None` to disable
+proxy_proto_header = `false` # set to `false` or `None` to disable
```

Comment:
Is saying `None` correct here? This implies you can write:

```toml
ip_header = None
proxy_proto_header = None
```

And it'll disable the header. But that isn't the case.

```suggestion
ip_header = "X-Real-IP" # set to `false` to disable
proxy_proto_header = `false` # set to `false` to disable
```